src/oge/emissions.py

```diff
@@ -542,16 +557,17 @@ def calculate_nox_from_fuel_consumption(
             x.fuel_consumed_mmbtu * x.controlled_non_ozone_season_nox_ef_lb_per_mmbtu,
         )
     )
-    # if there were not season-specific rates, fill in using the annual rate if available
+    # if there were not season-specific rates, fill in using the annual rate if
+    # available
     gen_fuel_allocated["controlled_nox_mass_lb"] = gen_fuel_allocated[
         "controlled_nox_mass_lb"
     ].fillna(
         gen_fuel_allocated["fuel_consumed_mmbtu"]
         * gen_fuel_allocated["controlled_annual_nox_ef_lb_per_mmbtu"]
     )
     # update the emision total using the controlled mass if available
-    gen_fuel_allocated["nox_mass_lb"].update(
-        gen_fuel_allocated["controlled_nox_mass_lb"]
+    gen_fuel_allocated.update(
+        pd.DataFrame({"nox_mass_lb": gen_fuel_allocated["controlled_nox_mass_lb"]})
```

Comment:
So this looks like a slightly different pattern than what Jeff was doing, which I think would be:
` gen_fuel_allocated.update({"nox_mass_lb": gen_fuel_allocated["controlled_nox_mass_lb"]}`

Does this need to be converted to a dataframe for this to work or would a dict work? Just out of curiosity.

Reply:
I will try. Keep you updated.

Reply:
It works. Chnging it.

Reply:
Done